app/src/main/java/io/apicurio/registry/ImportLifecycleBean.java

```diff
@@ -31,6 +33,7 @@ public class ImportLifecycleBean {
     RegistryStorage storage;
 
     @ConfigProperty(name = "registry.import.url")
+    @Info( category = "import", description = "The import URL", availableSince = "2.1.0.Final")
     Optional<URL> registryImportUrlProp;
```

Comment:
@EricWittmann Is this the registry data import URL, i.e., including full path to generated `.zip` file, or something else?

Reply:
IIRC that is a URL to a .zip file which was previously exported from a registry.  The point of this feature is to automatically pre-load some artifacts into the registry on startup.